lib/SPIRV/libSPIRV/SPIRVEntry.cpp

```diff
@@ -679,4 +679,19 @@ SPIRVType *SPIRVTypeStructContinuedINTEL::getMemberType(size_t I) const {
   return static_cast<SPIRVType *>(SPIRVEntry::getEntry(Elements[I]));
 }
 
+void SPIRVModuleProcessed::validate() const {
+  assert(WordCount == FixedWC + getSizeInWords(ProcessStr));
```

Comment:
I suggest to emit an actual error instead of assertion which is wiped out in release build.
Or, add some clarifying message at least.

Reply:
I think the plain assertion should be enough here, because if it fails, it means that we have malformed module, which is likely to be "reported" (in form of a segfault probably) somewhere else. I.e. this is a simple enough assumption which don't deserve a runtime check, IMHO.

Reply:
Okay, can we give this assert a message then?